docs/instrumenten/algoritmeregister.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Algoritmeregister
+toelichting: Het Algoritmeregister is de plek waar overheden informatie over hun algoritmes kunnen publiceren.
+vereiste:
+- algoritmeregister
+levenscyclus:
+- monitoring-en-beheer
+bouwblok:
+- transparantie
+rollen:
+- projectleider
+- gemandateerd-verantwoordelijke
+- privacy-officer
+- proceseigenaar
+- data-scientist
```

Comment:
moet de data scientist is met het register..?

Reply:
nee je hebt gelijk, die haal ik hier weg!